src/chttpd_handlers.erl

```diff
@@ -51,6 +54,20 @@ design_handler(HandlerKey, DefaultFun) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
+is_disabled(KeyBin) ->
+    BlacklistConfig = config:get("chttpd", "url_handler_blacklist", []),
```

Comment:
there is a point to be made to differentiate the different levels at which these can be blacklisted to mirror the different levels at which these can be registered:

```
[httpd_design_handlers]
[httpd_global_handlers]
[httpd_db_handlers]
```

Although I’d still prefer to have an actual list, as in:

```
[chttpd]
httpd_design_handlers = []
httpd_global_handlers = []
httpd_db_handlers = []
```